bitnami/grafana-operator/templates/deployment.yaml

```diff
@@ -93,18 +93,9 @@ spec:
           args: {{- include "common.tplvalues.render" (dict "value" .Values.operator.args "context" $) | nindent 12 }}
           {{- else }}
           args:
-            - --grafana-image={{ include "grafana-operator.grafana.baseImage" . }}
-            - --grafana-image-tag={{ .Values.grafana.image.tag | trim }}
-            - --grafana-plugins-init-container-image={{ include "grafana-operator.grafana.pluginsInitContainerImage" . }}
-            - --grafana-plugins-init-container-tag={{ default .Values.grafana.image.tag .Values.grafana.pluginsInitContainerImage.tag | trim }}
             {{- if .Values.operator.leaderElect }}
             - --leader-elect=True
             {{- end }}
-            {{- if (and .Values.operator.scanAllNamespaces (not .Values.operator.scanNamespaces)) }}
-            - --scan-all=True
-            {{- else if (and .Values.operator.scanNamespaces (not .Values.operator.scanAllNamespaces )) }}
-            - --namespaces={{ include "grafana-operator.joinListWithComma" .Values.operator.scanNamespaces }}
```

Comment:
It seems we are not using this helper any longer, do you think we should removed?